### DISCUSSION TITLE
Pimcore X workflow designer

### DISCUSSION THREAD

**Discussion by nirbanc2020** [pimcore/pimcore#11306] [General]
I have installed Pimcore X Workflow Designer and created workflow definition,but it is not saving. I am getting the following message:

Unrecognized option "custom_extensions" under "pimcore.workflows.wfdesigner_test_workflow". Available options are "audit_trail", "enabled", "globalActions", "initial_markings", "label", "marking_store", "placeholders", "places", "priority", "support_strategy", "supports", "transitions", "type".

Server threw exception - could not perform action. Please reload the admin interface and try again.
![kkkk](https://user-images.githubusercontent.com/73533273/151746094-323fe60a-ac14-4599-acef-bdcec22e5bd1.png)

**Comment by fashxp** [comment]
what pimcore version are you using?

**Reply by nirbanc2020** [reply]
Pimcore version 10

**Reply by fashxp** [reply]
you should use at least 10.2  ... and there this custom_extensions node should be defined.

**Reply by nirbanc2020** [reply]
Where I should define the custom_extensions node?

**Reply by fashxp** [reply]
you don't need to define anything. starting with pimcore 10.2, pimcore should define this configuration option (see [here](https://github.com/pimcore/pimcore/blob/v10.2.0/bundles/CoreBundle/DependencyInjection/Configuration.php#L1549)) thus your error message should never appear.

**Reply by nirbanc2020** [reply]
Thanks a lot..
